Compilation error when missing usb.h

Bisection narrows this down to commit 7420382 (a merge from upstream, I think). The code in `arm-jtag-ew.c` fails to compile because of a missing `usb.h`:
```
libtool: compile:  gcc -DHAVE_CONFIG_H -I. -I.. -I../src -I./src -I../src/helper -DPKGDATADIR=\"/usr/local/share/openocd\" -DBINDIR=\"/usr/local/bin\" -I../jimtcl -I./jimtcl -isystem /usr/include/libusb-1.0 -I/usr/include/libftdi1 -I/usr/include/libusb-1.0 -I./src/jtag/drivers/libjaylink/libjaylink -I../src/jtag/drivers/libjaylink -Wall -Wstrict-prototypes -Wformat-security -Wshadow -Wextra -Wno-unused-parameter -Wbad-function-cast -Wcast-align -Wredundant-decls -Wpointer-arith -Werror -g -O2 -MT src/jtag/drivers/libocdjtagdrivers_la-arm-jtag-ew.lo -MD -MP -MF src/jtag/drivers/.deps/libocdjtagdrivers_la-arm-jtag-ew.Tpo -c ../src/jtag/drivers/arm-jtag-ew.c -o src/jtag/drivers/libocdjtagdrivers_la-arm-jtag-ew.o
../src/jtag/drivers/arm-jtag-ew.c:26:10: fatal error: usb.h: No such file or directory
   26 | #include <usb.h>
      |          ^~~~~~~
compilation terminated.
```

Looking at the history, I think the correct fix is just to remove that include line (no longer needed since dropping libusb0 support?). I'll make a PR doing so, but wanted an issue to point to, in case this is the wrong fix.

--- a/src/jtag/drivers/arm-jtag-ew.c
+++ b/src/jtag/drivers/arm-jtag-ew.c
@@ -23,7 +23,6 @@
 #include <jtag/interface.h>
 #include <jtag/commands.h>
 #include "helper/system.h"
-#include <usb.h>
 #include "libusb_helper.h"
 
 #define USB_VID						0x15ba

